telegram_recorder.cpp

```diff
@@ -338,7 +349,12 @@ void TelegramRecorder::retrieveAndWriteUserFromTelegram(td_api::int53 userID) {
       TelegramUser* user = new TelegramUser;
       user->userID = u->id_;
       user->fullName = (u->last_name_ == "" ? u->first_name_ : (u->first_name_ + " " + u->last_name_));
-      user->userName = u->username_;
+      auto usernames = join(u->usernames_->active_usernames_);
+      auto disabled_usernames = join(u->usernames_->disabled_usernames_);
+      if (!disabled_usernames.empty())  {
```

Comment:
I would leave the `disabled_usernames` on a separate field in the `TelegramUser` class, with the idea of adding it to a new column in the sqlite schema. Should be easier to query the info.